src/lib.rs

```diff
@@ -196,12 +196,12 @@ impl<T: Ord> OrderedCollection<T> {
     // we therefore need to find at what depth a single prefetch fetches all the descendants.
     // it turns out that, at depth k under some node with index i, the leftmost child is at:
     //
-    //   2^k * i + 2^(k-1) + 2^(k-2) + ... + 2^0 = 2^k * i + 2^k - 1
+    //   2^k * i
```

Comment:
Maybe add a link here (in the comment) back to the PR to indicate that we're diverging from the original paper version?

Reply:
done